script/Ciel/Day 9/09_01_CIEL09_4_1.txt

```diff
@@ -187,23 +214,28 @@ As long as I can't come up with a clear answer, I can't talk to Senpai.
 [sha:4d77982aee2573503299a9dd7278e8b075749b0d]{
 -- Page 489, Offset 23398.
 -- それは先輩から出たとは思えない、あからさまな侮蔑だった。
-I would've never expected to hear such contempt from Senpai.
+I would've never expected to hear such unbridled contempt coming from Senpai.
```

Comment:
My bigger problem is with constructing it like that (with 'I' as the subject). It makes it seem like it's mostly about Shiki reacting to her 侮蔑 rather than a description of her doing it.

Reply:
I mean, this is all told from Shiki's perspective. He's doing the 思えない. Even in your example, he's the one thinking it's seems out of character. I think that English is just a much more egotistical language, and while you can construct it a bit more passively it feels a lot more J->E that way imo.

Reply:
Well, of course he's the one thinking it, he's the one thinking everything; it's his perspective.
But that doesn't mean every sentence has to hinge on using "I" as the subject.
Shiki can equally formulate a thought using Ciel as a subject, then his comments on her will implicitly be his thoughts on the matter.

There's just something about this one that very much bothers me. It feels like it completely shifts the focus from あからさまな侮蔑 to 先輩から出たとは思えない, when the former is the subject of the sentence. In the original 先輩から出たとは思えない is in service of あからさまな侮蔑.

Ah well, I can't come up with any other better sentences than the one I gave.

Reply:
Looking at it again came up with something non-I that I'm ok with.